python/ray/serve/tests/test_proxy.py

```diff
@@ -547,6 +550,66 @@ async def test_proxy_asgi_receive(self):
 
         queue.close.assert_called_once()
 
+    @pytest.mark.asyncio
+    @pytest.mark.parametrize(
+        "disconnect",
```

Comment:
Nitpick (non-blocker): We can probably refactor this to pass a tuple of received messages and expected messages so we don't need to do have the if-else statement below. Just an idea, no strong feeling either way.